docs/feature-flagging/dynamic-config.md

```diff
@@ -10,6 +10,10 @@ This is achieved by the engineering team implementing a number of variables in t
 
 Once implemented, Dynamic Configuration allows non-technical users to set values in Eppo's UI and see the change live on site in seconds without a deploy. In particular, a marketing user can set different variants with different values, allowing them to run multiple tests all from the same flag without any additional engineering work. This can be used to update or test headline copy, CTA button copy and color, and image assets used.
```

Comment:
```suggestion
Once implemented, Dynamic Configuration allows non-technical users to set values in Eppo's UI and see the change live on site in seconds without a deployment. In particular, a marketing user can set different variants with different values, allowing them to run multiple tests all from the same flag without any additional engineering work. This can be used to update or test headline copy, CTA button copy and color, and image assets used.
```